Modules/_ssl.c

```diff
@@ -3576,7 +3582,7 @@ set_maximum_version(PySSLContext *self, PyObject *arg, void *c)
     return set_min_max_proto_version(self, arg, 1);
 }
 
-#ifdef TLS1_3_VERSION
+#if defined(TLS1_3_VERSION) && !defined(OPENSSL_NO_TLS1_3)
```

Comment:
Are tickets used only for post-handshake authentication? From the docs it looks like a separate TLSv1.3 feature.

Reply:
No, tickets are used for session resumption and are unrelated to post-handshake authentication. I believe `get_num_tickets` pertains to [a post-handshake message added in TLSv1.3](https://www.rfc-editor.org/rfc/rfc8446.html#section-4.6.1). `SSL_CTX_get_num_tickets`'s [docs](https://www.openssl.org/docs/manmaster/man3/SSL_CTX_get_num_tickets.html) indicate as much.

This change just updates that guard to honor [OpenSSL's `OPENSSL_NO_TLS1_3`](https://github.com/search?q=repo%3Aopenssl%2Fopenssl+OPENSSL_NO_TLS1_3&type=code) as [other parts](https://github.com/python/cpython/blame/92893fd8dc803ed7cdde55d29d25f84ccb5e3ef0/Modules/_ssl.c#L253) of `_ssl.c` already do.